lib/fizzy/parser_expr.cpp

```diff
@@ -861,11 +861,12 @@ parser_result<Code> parse_expr(const uint8_t* pos, const uint8_t* end, FuncIdx f
 
             uint32_t offset;
             std::tie(offset, pos) = leb128u_decode<uint32_t>(pos, end);
-            push(code.immediates, offset);
+            code.instructions.push_back(opcode);
```

Comment:
Btw, why is the old code (end of loop) using `emplace_back(opcode)` but the new commits are not?